stats/stats/src/charts/counters/total_txns.rs

```diff
@@ -68,7 +67,7 @@ mod tests {
     #[tokio::test]
     #[ignore = "needs database to run"]
     async fn update_total_txns() {
-        let counter = TotalTxns::default();
-        simple_test_counter("update_total_txns", counter, "17").await;
+        let counter = TotalTxns::new(Default::default());
+        simple_test_counter("update_total_txns", counter, "16").await;
```

Comment:
why did it change?

Reply:
It changed because we used to count like `count(transactions)`, now we apply filter that `transaction.block.consensus = true`